src/quo/components/wallet/token_input/style.cljs

```diff
@@ -16,27 +16,56 @@
    :flex-direction     :row
    :justify-content    :space-between})
 
+(defn token-name
+  [theme]
+  {:color          (colors/theme-colors colors/neutral-50 colors/neutral-40 theme)
+   :margin-right   8
+   :padding-bottom 2})
+
+(def token-label-container
+  {:position       :absolute
+   :left           32
+   :right          0
+   :bottom         0
+   :top            0
+   :flex-direction :row
+   :align-items    :flex-end})
+
+(def text-input-container
+  {:position :absolute
+   :top      0
+   :bottom   0
+   :left     32 ; token image size
+   :right    0})
+
+(def text-input-dimensions
+  (assoc typography/heading-1
+         :font-weight  "600"
+         :margin-left  8
+         :margin-right (if platform/ios? 6 4)
+         :padding      0
+         :height       "100%"))
+
```

Comment:
I am not sure all these changes is necessary. Ref: https://github.com/status-im/status-mobile/pull/18522#pullrequestreview-1822742080

Reply:
Thank you @ulisesmac for the clarification. Closed my PR.